src/backups2datalad/datasetter.py

```diff
@@ -600,9 +600,16 @@ async def debug_logfile(self, quiet_debug: bool) -> None:
                 h.setLevel(screen_level)
         # Superdataset must exist before creating anything in the directory:
         await self.ensure_superdataset()
-        logdir = self.config.dandiset_root / ".git" / "dandi" / "backups2datalad"
-        logdir.mkdir(exist_ok=True, parents=True)
         ts = datetime.now(timezone.utc)
+        logdir = (
+            self.config.dandiset_root
+            / ".git"
+            / "dandi"
+            / "backups2datalad"
+            / f"{ts:%Y}"
+            / f"{ts:%m}"
```

Comment:
neat! I didn't know that we can use those format directives directly in f-strings! may be it was just me though ;) attn @candleindark @asmacdo